backend/tests/conftest.py

```diff
@@ -258,3 +253,25 @@ def _helper():
         window.later = datetime.datetime.utcnow() + datetime.timedelta(seconds=1)
 
     return _helper
+
+
+@fixture
+def tweak_settings():
+    """
+    Provide a fixture to use as a context manager where the app settings may be temporarily changed.
+    """
+
+    @contextlib.contextmanager
+    def _helper(**kwargs):
+        """
+        Context manager for tweaking app settings temporarily.
+        """
+        previous_values = {}
+        for (key, value) in kwargs.items():
+            previous_values[key] = getattr(settings, key)
+            setattr(settings, key, value)
+        yield
+        for (key, value) in previous_values.items():
+            setattr(settings, key, value)
+
+    return _helper
```

Comment:
Why is this fixture needed?

Reply:
It's being used (right now) the multi-tenancy unit test (which I originally forgot to add to my commit). It will also be nice to have when we write additional unit tests that depend on certain environment settings. This is copy-pasta from jobbergate.

Reply:
Since the test was missing from the first commit I was wondering where it was being used. Now I understand it, thanks!